### PR TITLE
implemented terminal stake info demo page to live

### PR DESCRIPTION
**Pull request by Sirmorrison**
image

### CODE REVIEW

**Review by ademuanthony — CHANGES_REQUESTED** [review]
I am experiencing a panic whenever I try to navigate to the `Stake info` page.
I am using decrlibwallet, the `walletrpcserver` is not set in the config

**Comment by Sirmorrison** [comment]
> 
> 
> I am experiencing a panic whenever I try to navigate to the `Stake info` page.
> I am using decrlibwallet, the `walletrpcserver` is not set in the config

File: terminal/terminal.go [hunk 1]
```diff
@@ -81,7 +80,7 @@ func terminalLayout(tviewApp *tview.Application, walletMiddleware app.WalletMidd
 			changePageColumn(pages.HistoryPage())
 		}).
 		AddItem("Stakeinfo", "", 'k', func() {
-			changePageColumn(pages.ReceivePage())
+			changePageColumn(pages.StakeinfoPage(walletMiddleware, setFocus, clearFocus))
```
Comment (ademuanthony) [inline terminal/terminal.go:83]: We need a way to return the focus back to the main menu from this page. I have to restart the application each time I want to go to the main menu

Comment (ademuanthony) [reply terminal/terminal.go:83]: I see that the Esc key works as we discussed.
But we will need to have a help-tip at the footer of every page that explains the control keys

**Review by ademuanthony — APPROVED** [review]

**Pull request merged**